src/RuleSet.php

```diff
@@ -230,7 +230,7 @@ final class RuleSet implements RuleSetInterface
             'switch_continue_to_break' => true,
             'trailing_comma_in_multiline_array' => true,
             'trim_array_spaces' => true,
-            'visibility_required' => true,
+            'visibility_required' => ['elements' => ['method', 'property']],
```

Comment:
How about adding visibility for constants as well ?

Reply:
That change is not to the PSR-12 preset.

Reply:
Oooh yes, sorry. Fair enough, I validate this :-)

Reply:
Those changes are there so that when/if const is added to the defaults later, the psr2 preset does not fix too much.

Reply:
We only change default values on major releases, so I think keeping the default is fine.